components/camel-tahu/src/test/resources/hivemq-ce/extensions/sparkplug-tck/coverage/coverage-sparkplug.html

```diff
@@ -0,0 +1,3105 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file (along with other `.png` and similar) needed?

Reply:
Strictly speaking, no. The only files required to support Sparkplug B TCK behavior in the HiveMQ broker are the config files and the `sparkplug-tck-3.0.0.jar` extension library. I included the others here as a complete unzip of the TCK extension to follow the TCK installation instructions.

I have removed the `coverage` directory.

Reply:
So, we cannot have binary jar files on the repo:

1. Licensing
2. Security 
3. Makes the checkout very slow